packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -15,9 +23,270 @@ class DiffPane extends StatefulWidget {
   State<DiffPane> createState() => _DiffPaneState();
 }
 
+/// Will notify if count of items or current index changed.
+class _DiffPaneController with ChangeNotifier {
+  _DiffPaneController() {
+    index.addListener(() => notifyListeners());
+  }
+
+  final scrollController = ScrollController();
+
+  /// The list contains one item that show information and all others
+  /// are snapshots.
+  final snapshots = <DiffListItem>[InformationListItem()];
+
+  final index = ValueNotifier<int>(0);
```

Comment:
nit: rename to `selectedIndex` so that it is clear what `index` we are referring to